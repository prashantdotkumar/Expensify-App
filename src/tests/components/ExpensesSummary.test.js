import React from "react";
import { shallow } from "enzyme";
import { ExpensesSummary } from "../../components/ExpensesSummary";

test("should correctly render expense summary with 1 expense", () => {
  const wrapper = shallow(
    <ExpensesSummary expenseCount={1} expenseTotal={235} />
  );
  expect(wrapper).toMatchSnapshot();
});

test("should correctly render expense summary with multiple expenses", () => {
  const wrapper = shallow(
    <ExpensesSummary expenseCount={10} expenseTotal={23512340987} />
  );
  expect(wrapper).toMatchSnapshot();
});
