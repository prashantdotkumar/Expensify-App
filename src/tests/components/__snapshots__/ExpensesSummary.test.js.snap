// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render expense summary with 1 expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totaling to 
    $0.00
  </h1>
</div>
`;

exports[`should correctly render expense summary with multiple expenses 1`] = `
<div>
  <h1>
    Viewing 
    10
     
    expenses
     totaling to 
    $0.00
  </h1>
</div>
`;
